product/reports/200_Red Hat - Reports/001_CPU Right Sizing Recommendations.yaml

```diff
@@ -0,0 +1,95 @@
+---
```

Comment:
My issue is still with the beginning of the filename, this would create a `Red Hat` folder for reports which I'm not sure how okay is in upstream...

Reply:
👍 ^^^ Need to get the **Red Hat** out of the file name for upstream